Song_sort_Title.css

```diff
@@ -67,62 +41,152 @@ li{
 }
 
 
+
 nav a{
 	display:block;
 	width:100px;
 	padding: 20px 15px;
 	text-decoration:none;
-	font-family:Arial;
-	color:black;
+	font-family: 'Open Sans', sans-serif;
+	font-weight:bold;
+	color:white;
 	text-align: center;
 }
 
 nav a:hover{
-	background:purple;
+	background:black;
 	transition:0.5s;
 }
 
-.content_1{
-	background-color: white;
+ul li ul{
+	padding: 0;
+	position: absolute;
+	top: 58px;
+	width: 100px;
+	display: none;
+	visibility: hidden;
+	z-index: 5;
+}
+	
+ul li ul li{
+	background-color:  #0C59CF;
+	display: block;
+}
+
+ul li ul li:hover{background: black;}
+
+ul li:hover ul{
+	display: block;
+	visibility: visible;
+}
+
+.logo img{
+	position:absolute;
+	margin-top:-20px;
+}
+
+
+
+.img_1{
 	grid-column-start: 1;
 	grid-column-end: span 4;
-	z-index: 2;	
```

Comment:
21/9: I remove to set height so the banner and and also remove the border. Also I remove the z-index because I don't think I will make some thing display over top of the banner.